src/gsy_e/models/strategy/forward/load.py

```diff
@@ -0,0 +1,83 @@
+from typing import Dict, TYPE_CHECKING
+
+from gsy_framework.enums import AvailableMarketTypes
+from pendulum import DateTime
+
+from gsy_e.constants import FLOATING_POINT_TOLERANCE
+from gsy_e.models.strategy.energy_parameters.energy_params_eb import (
+    ConsumptionStandardProfileEnergyParameters)
+from gsy_e.models.strategy.forward import ForwardStrategyBase
+from gsy_e.models.strategy.forward.order_updater import OrderUpdaterParameters
+
+if TYPE_CHECKING:
+    from gsy_e.models.market.forward import ForwardMarketBase
+    from gsy_framework.data_classes import Trade
+    from gsy_e.models.state import LoadState
+
+
+class ForwardLoadStrategy(ForwardStrategyBase):
+    """
+    Strategy that models a Load that trades with a Standard Solar Profile on the forward
+    exchanges.
+    """
+    def __init__(
+            self, capacity_kW: float,
+            order_updater_parameters: Dict[AvailableMarketTypes, OrderUpdaterParameters]):
+        super().__init__(order_updater_parameters)
+        self._energy_params = ConsumptionStandardProfileEnergyParameters(capacity_kW)
+
+    @property
+    def state(self) -> "LoadState":
+        return self._energy_params.state
+
+    def event_activate(self, **kwargs):
+        self._energy_params.event_activate_energy(self.area)
+
+    def remove_open_orders(self, market: "ForwardMarketBase", market_slot: DateTime):
+        bids = [bid
+                for bid in market.slot_bid_mapping[market_slot]
+                if bid.buyer == self.owner.name]
+        for bid in bids:
+            market.delete_bid(bid)
+
+    def post_order(self, market: "ForwardMarketBase", market_slot: DateTime):
+        order_rate = self._order_updaters[market][market_slot].get_energy_rate(
+            self.area.now)
+        capacity_percent = self._order_updaters[market][market_slot].capacity_percent / 100.0
+        max_energy_kWh = self._energy_params.peak_energy_kWh * capacity_percent
+        available_energy_kWh = self._energy_params.get_available_energy_kWh(
+            market_slot, market.market_type)
+
+        posted_energy_kWh = self._energy_params.get_posted_energy_kWh(
+            market_slot, market.market_type)
+
+        available_energy_kWh = min(available_energy_kWh - posted_energy_kWh, max_energy_kWh)
```

Comment:
I propose not to re-use the `available_energy_kWh` but to name it `order_energy` or similar. What do you think?

Reply:
Agreed, fixing